src/utils/Api.js

```diff
@@ -0,0 +1,54 @@
+import { processServerResponse } from "./utils";
+
+const baseUrl =
+  process.env.NODE_ENV === "production"
+    ? "https://api.mynewsexplorer.jumpingcrab.com"
+    : "http://localhost:3002";
+
+function getUserArticles(token) {
+  return fetch(`${baseUrl}/articles`, {
+    method: "GET",
+    headers: {
+      Accept: "application/json",
+      "Content-Type": "application/json",
+      Authorization: `Bearer ${token}`,
+    },
+  }).then(processServerResponse);
```

Comment:
**COULD BE IMPROVED**

You can make a special function for fetching and checking responses **not to duplicate** it in every request:

```js

  function request(url, options) {
    return fetch(url, options).then(processServerResponse)
  }

```

And now you can safely replace `fetch` with `request` and remove repeated checking. And you don’t need to change anything else.